shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -131,6 +133,10 @@ class FlutterPlatformViewsController {
   GrContext* overlays_gr_context_;
   SkISize frame_size_;
 
+  // This is the number of frames the task runners will stay
+  // merged after a frame where we see a mutation to the embedded views.
+  static const int kDefaultMergedLeaseDuration = 10;
```

Comment:
Say something about how you picked this number (even if it's completely arbitrary would be useful to record that for the next person coming to revisit this number to know)

Reply:
Done